Parsers/ASimAuthentication/ARM/ASimAuthenticationAADNonInteractive/README.md

```diff
@@ -1,15 +1,18 @@
-# Azure active directory nonInteractive signin logs ASIM Authentication Normalization Parser
+# Microsoft AAD (nonInteractive signin logs) ASIM Authentication Normalization Parser
+
+This template deploys the ASIM Authentication schema parser for Microsoft AAD (nonInteractive signin logs).
```

Comment:
Another YAML fix 
- "nonInteractive signin logs" -> "non-interactive authentication"
- To be consistent, use authentication instead of "signin logs", which is the way the interactive and service principal title and description are.